src/models/generated/SupplementalStaticReserve.jl

```diff
@@ -0,0 +1,80 @@
+#=
+This file is not auto-generated. Do edit if needed.
+=#
+"""
+    mutable struct SupplementalStaticReserve{T <: ReserveDirection} <: Reserve{T}
```

Comment:
Do we still need to add this additional reserve object? It seems that the expected implementation is not different from the current static reserves.

Reply:
It's exactly the same you are right, but I added this additional reserve so we can have both in the same system. Is it already possible to have multiple static reserves with different formulations?